src/hooks/admin/useAdminProducts.tsx

```diff
@@ -115,12 +115,46 @@ function useAdminProducts(workspaceId: string | undefined) {
   };
 
   const addCategories = (name: string) => {
-    adminApi.post('/product-category', { name, workspaceId }).catch((error) => {
-      console.error('Failed to add products categories : ', error);
-    });
+    adminApi
+      .post('/product-category', { name, workspaceId })
+      .then(() => fetchCategories())
+      .catch((error) => {
+        console.error('Failed to add products categories : ', error);
+      });
   };
 
-  return { fetchProducts, addProduct, fetchCategories, addCategories, deleteProducts, fetchProduct, editProduct, editProductSellable };
+  const reorderCategories = (categoryIds: number[]) => {
```

Comment:
```suggestion
  const reorderCategories = (productCategoryIds: number[]) => {
```
로 바꿔서 아래 workspaceId처럼 줄여서 넣을 수 있게 하시죠~